packages/webapp/package.js

```diff
@@ -1,12 +1,12 @@
 Package.describe({
   summary: "Serves a Meteor app over HTTP",
-  version: "2.0.5",
+  version: "2.0.6",
 });
 
 Npm.depends({
   "cookie-parser": "1.4.6",
-  express: "5.0.1",
-  "@types/express": "5.0.0",
+  express: "5.1.0",
+  "@types/express": "5.0.1",
```

Comment:
This PR also updates to Express 5.1.0, not just a Meteor docs improvements. It's fine to include it in 3.3, thanks for that. 🙏

In the future, please send a separate PR when upgrading tool versions in Meteor core packages.